Common/QuantConnect.csproj

```diff
@@ -256,6 +256,7 @@
     <Compile Include="Securities\Equity\EquityMarginModel.cs" />
     <Compile Include="Securities\Equity\EquityPortfolioModel.cs" />
     <Compile Include="Securities\DelayedSettlementModel.cs" />
+    <Compile Include="Securities\Equity\PatternDayTraderMarginModel.cs" />
```

Comment:
We should name this file the same as the class name: `PatternDayTradingMarginModel`